### DISCUSSION TITLE
WordPress Theme

### DISCUSSION THREAD

**Discussion by peiche** [codyhouse/codyhouse-framework#85] [Show and tell]
I've created a WordPress theme using CodyFrame:

### [Garrick](https://github.com/peiche/garrick)

There is built-in support for dark mode and custom accent colors, thanks to CF's extensive use of custom properties, and fallback IE support. Post listing views (like the blog page, search results, and archives) support single column or grid views. It also includes support for a wide range of Google fonts.

The theme is built using Justin Tadlock's [Mythic](https://github.com/justintadlock/mythic) starter theme, making it modular in nature. Any theme view (located in `/resources/views`) can be replaced in a child theme. The base theme does not include any CF Components (per the license), but its modularity makes replacing individual components easy. The theme also includes 36 Nucleo icons in SVG format.

----

My [website](https://eichefam.net) is running a child theme of Garrick, but I can't make its source public due to the Components license terms. If there is interest, I could write up a how-to for replacing a page fragment with a CF Component in a child theme.

**Comment by sebastiano-guerriero** [comment]
That's very interesting! I'll check it out 👍

**Comment by peiche** [comment]
I'm happy to announce that Garrick now supports CodyFrame 3.0! I made the decision to drop the fallback stylesheet and embrace the gap property; its custom block styles use it instead of negative margins (yuck).

Edit: the latest release can be found [here](https://github.com/peiche/garrick/releases/tag/v1.5.0).

**Reply by sebastiano-guerriero** [reply]
Great job!
